bokehjs/src/coffee/core/selection_manager.coffee

```diff
@@ -55,14 +56,20 @@ export class SelectionManager extends HasProps
 
   inspect: (tool, renderer_view, geometry, data) ->
     source = @source
-    if source != renderer_view.model.data_source
-      logger.warn('inspect called with mis-matched data sources')
 
-    view = renderer_view.model.view
-    if source != view.source
-      logger.warn('inspect called with view and data source mismatch')
+    # if source != renderer_view.model.data_source
+    #   logger.warn('inspect called with mis-matched data sources')
+    #
+    # view = renderer_view.model.view
+    # if source != view.source
+    #   logger.warn('inspect called with view and data source mismatch')
 
-    indices = renderer_view.model.view.convert_selection_from_subset(renderer_view.hit_test(geometry))
+    hit_test_result = renderer_view.hit_test(geometry)
```

Comment:
I'm currently unable to get the graph nodes to be properly "hit". It's always returning "empty" hit test results